src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -670,7 +670,23 @@ def apply_sbp_treatment_ramp(
         # Generate other useful helper indexes
         low_sbp = measured_sbp[measured_sbp < data_values.SBP_THRESHOLD.LOW].index
         high_sbp = measured_sbp[measured_sbp >= data_values.SBP_THRESHOLD.HIGH].index
-        newly_prescribed = overcome_initial_prescription_inertia.difference(
+        medicated_high_sbp = currently_medicated.intersection(high_sbp)
+
+        # Update inertia values for those who moved up a medication level on the previous time step
```

Comment:
This comment isn't actually correct. It's "those who moved up a medication level last time they had a high SBP measured during a doctor visit"